bedrock/firefox/templates/firefox/whatsnew/whatsnew-mobile-de.html

```diff
@@ -0,0 +1,73 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import send_to_device with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox.mobile.index') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew-mobile-de{% endblock %}
+{% block body_class %}{{ super() }}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_mobile_de') }}
+{% endblock %}
+
+{% block content %}
+<main role="main" class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content">
+    <img class="wnp-main-image" src="{{ static('protocol/img/logos/firefox/browser/logo-md.png') }}" alt="">
```

Comment:
Reviewing the rendered HTML, I'm not getting the [high-res version](https://github.com/mozilla/protocol-assets/blob/main/logos/firefox/logo-md-high-res.png) of the hero logo. Is it missing a `data-high-res="true"` attribute? 

An alternative would be to to use [the SVG version](https://github.com/mozilla/protocol-assets/blob/main/logos/firefox/logo.svg), and make sure the `.wnp-main-image` class has a max-width.